rapids-cmake/export/export.cmake

```
@@ -80,11 +80,14 @@ calls to :cmake:command:`find_dependency`, or :cmake:command:`CPMFindPackage`.
   Optional value to specify what namespace all targets from the
   EXPORT_SET will be placed into. When provided must match the pattern
   of `<name>::`.
-  If not provided all targets will be placed in the `<project_name>::`
-  namespace
+  A recommended namespace could be `<project_name>::`.
+  If not provided, no namespace is used.
 
-  Note: When exporting with `BUILD` type, only `GLOBAL_TARGETS` will
-  be placed in the namespace.
+  Note:
+  - When exporting with `BUILD` type, only `GLOBAL_TARGETS` will be placed
+  in the namespace.
+  - The namespace can be configured on a per-target basis instead using the
+  EXPORT_NAME property.
```

```suggestion
  :cmake:prop_tgt:`EXPORT_NAME <cmake:prop_tgt:EXPORT_NAME>`  property.
```
So we cross reference the official docs for `EXPORT_NAME`